src/styles/resources.scss

```diff
@@ -4,10 +4,31 @@
     margin-left: 5px;
     margin-bottom: 25px;
     cursor: pointer;
+    padding: 25px 0 0 25px;
+    font-weight: bold;
+    font-size: 30px;
+    color: white;
+    line-height: 33px;
   }
 
+
   &__rounded-card {
     border-radius: 18px;
     cursor: pointer;
+    background-color: #386A3F;
+    min-height: 100%;
```

Comment:
could you try giving the rounded-card a top-bottom margin? Wondering if that would help with the spacing on mobile